addons/website_sale/views/templates.xml

```diff
@@ -1117,9 +1117,11 @@
                                             </div>
                                         </section>
                                     </div>
-                                    <t t-if="is_view_active('website_sale.product_tags')" t-call="website_sale.product_tags">
-                                        <t t-set="all_product_tags" t-value="product_variant.all_product_tag_ids"/>
-                                    </t>
+                                    <div class="mt-2">
+                                        <t t-if="is_view_active('website_sale.product_tags')" t-call="website_sale.product_tags">
+                                            <t t-set="all_product_tags" t-value="product_variant.all_product_tag_ids"/>
+                                        </t>
+                                    </div>
```

Comment:
```suggestion     
                                    <div t-if="is_view_active('website_sale.product_tags')" t-call="website_sale.product_tags" class="mt-2">
                                         <t t-set="all_product_tags" t-value="product_variant.all_product_tag_ids"/>
                                    </div>
```

Also do not remove above code instead make it invisible with comment `remove me in master`  and make it invisible to avoid breaking customer xpath

Reply:
Done the changes.
I'm sorry, but I don't understand how I can make it invisible to the t tag so it doesn't affect future customization.

Reply:
Not future customization, Other customers who might did customization and use this line as xpath like ps-tech or offshore team for their customer